data/data/powervs/iaas/power-iaas.tf

```diff
@@ -0,0 +1,22 @@
+provider "ibm" {
+ ibmcloud_api_key = var.ibmcloud_api_key
+}
+
+data "ibm_resource_group" "group" { 
+  name = var.cloud_resource_group
+}
+
+resource "ibm_resource_instance" "resource_instance" { 
+  name     = var.service_instance_name
+  service  = "power-iaas"
+  plan     = var.cloud_instance_id
+  location = var.ibmcloud_region
+  tags     = var.service_tags 
+  resource_group_id = data.ibm_resource_group.group.id 
+  
+  timeouts { 
```

Comment:
i think the installer folks will want these to be hard-coded, so i'd go head and hard-code them here. i at least don't remember seeing a configurable timeout this granular for any of the other platforms. iirc when i tried this, it took about 2 mins to create it and 1 to delete it. not sure what you've seen but whatever you think is reasonable is fine for now

Reply:
I believe 10m was in the example, and 15m came from your example... Whatever timeouts work for me may not work for others, so I left it at 15m, and allowed for it to be a variable so we can extend it in the future if we have issues.  Are you certain you'd like to hard code this instead of just defaulting to something?

Reply:
i wasn't referring to the example. i meant the real-time experiments we both did. i can't remember how long it really took. are you saying that took yours closer to 15? and yes, i think we can safely hard-code to something for this particular timeout. not some of the other things in the install, though.